ConsoleApp/Program.cs

```diff
@@ -0,0 +1,143 @@
+﻿struct StringAnalyzer
+{
+    public int numberOfLetters;
```

Comment:
we haven't learnt it, but name convention for public field is pascal case, just for you to know :)
```suggestion
    public int NumberOfLetters;
```